main/src/library/IOError.flix

```diff
@@ -13,6 +13,38 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+// TODO: Replace IOError with IoError once File and Files are removed
```

Comment:
We try not to leave TODOs in source code. Instead we track on GitHub, so just remove this.